src/types/common.ts

```diff
@@ -79,3 +81,5 @@ export interface CustomEventHook<T> {
 export interface CustomEventHookWrapper<T> extends Event {
     detail: CustomEventHook<T>
 }
+
+export type GetPluginApi = (uuid: string) => SettersForExtensibleUIs
```

Comment:
Make it private where it's used, no need to be in a shared file, as it's just the method signature.